models/DataObject/Classificationstore/CollectionConfig.php

```diff
@@ -247,6 +294,20 @@ public function getCreationDate()
         return $this->creationDate;
     }
 
+    /**
+     * Returns all group belonging to this collection
+     *
+     * @return CollectionGroupRelation[]
+     */
+    public function getRelations()
+    {
+        $list = new CollectionGroupRelation\Listing();
+        $list->setCondition('colId = ' . $this->id);
+        $list = $list->load();
+
+        return $list;
+    }
+
```

Comment:
This code is not really related to this PR, so we should put it into another PR targeting `10.x`.

Reply:
You're right I will raise another PR for this. Removed...